client/profile-wizard/index.js

```diff
@@ -87,6 +91,11 @@ class ProfileWizard extends Component {
 			step: this.getCurrentStep().key,
 		} );
 
+		updateProfileItems( {
```

Comment:
I think this call can be removed.  Once they update `skipped` to `true` I don't think we need to revert it back to `false`.

It can remain `true` and they either complete the profiler or not.  Leaving `skipped` as is provides us more information.

Reply:
Sounds good to me. I removed it in the commit 4b73e58